elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -38,18 +44,19 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
     border-bottom: pfe-local(BorderBottom, $region: panel);
     border-left: pfe-local(BorderLeft, $region: panel);
 
-    padding: pfe-local(Padding, $region: panel); //16px
-
     @media screen and (min-width: pfe-breakpoint(md)) {
-      padding: calc(#{pfe-local(Padding, $region: panel)} * 2);
+      //padding: calc(#{pfe-local(Padding, $region: panel)} * 2);
     }
 
     @media screen and (min-width: pfe-breakpoint(lg)) {
-      padding: calc(#{pfe-local(Padding, $region: panel)} * 3);
+      //padding: calc(#{pfe-local(Padding, $region: panel)} * 3);
     }
 
     @media screen and (min-width: pfe-breakpoint(xl)) {
-      padding: calc(#{pfe-local(Padding, $region: panel)} * 4);
+      --pfe-tabs__panel--PaddingTop: pfe-local(PaddingTop, $region: panel, $fallback: calc(#{pfe-var(container-spacer)} * 3));
```

Comment:
A general goal of all PFE components is to leave the CSS variables empty if possible, and lean on the fallback values. That way other developers and projects can leverage these "hooks" and customize if needed. If they already have values, they are harder (or sometimes impossible) to override.